u_common.go

```diff
@@ -164,6 +182,7 @@ var (
 	HelloChrome_96   = ClientHelloID{helloChrome, "96", nil}
 	HelloChrome_100  = ClientHelloID{helloChrome, "100", nil}
 	HelloChrome_102  = ClientHelloID{helloChrome, "102", nil}
+	HelloChrome_106  = ClientHelloID{helloChrome, "106", nil}
```

Comment:
Since `Chrome_106` shares the fingerprint with `Chrome_102`, let's keep `102` and get rid of `106`.